portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro

```diff
@@ -687,7 +687,9 @@ definition {
 	}
 
 	@summary = "Default summary"
-	macro viewAutocompleteSuggestions(notPresent = null, autocompleteText = null) {
+	macro viewAutocompleteSuggestions(notPresent = null,autocompleteText = null) {
+		var key_autocompleteText = ${autocompleteText};
+
```

Comment:
Hey Vini, isn't the autocompleteText variable required for this function to work? if yes, I believe it shouldn't be null.

Reply:
Oh yeah, you're right. I think this was put by SF and I didn't remember to check. Thanks, Manu 🕵🏽‍♀️

Reply:
Thank you!